index.js

```diff
@@ -33,4 +33,29 @@ app.get('/api/courses/:id', (req, res) => {
     if (!courses) return res.status(404).send('The course was not found');
     
     res.send(courses);
-});
\ No newline at end of file
+});
+
+// Update / Edit Course
+app.put('/api/courses/:id',(req,res) => {
+    const course = courses.find(c=>c.id === parseInt(req.params.id));
```

Comment:
I just read your file and have a suggestion, how about this `const courses = dB.find(c => c.id === parseInt(req.params.id));` to read from file system.

Reply:
fixed.